data/categories/sg_sporting_goods.yml

```diff
@@ -6935,6 +6937,167 @@
   attributes:
   - color
   - pattern
+- id: sg-4-1-10
+  name: Wingfoils
+  children:
+  - sg-4-1-10-1
+  - sg-4-1-10-2
+  - sg-4-1-10-3
+  - sg-4-1-10-4
+  - sg-4-1-10-5
+  attributes: []
+- id: sg-4-1-10-1
+  name: Wingfoil Wings
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-10-2
+  name: Wingfoil Foils
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-10-3
+  name: Wingfoil Parts
+  children:
+  - sg-4-1-10-3-1
+  - sg-4-1-10-3-2
+  attributes: []
+- id: sg-4-1-10-3-1
+  name: Wing Parts
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+- id: sg-4-1-10-3-2
+  name: Foil Parts
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+- id: sg-4-1-10-4
+  name: Wingfoil Accessories
+  children:
+  - sg-4-1-10-4-1
+  - sg-4-1-10-4-2
+  attributes: []
+- id: sg-4-1-10-4-1
+  name: Wingfoil Carry Bags & Covers
+  children: []
+  attributes:
+  - bag/case_features
+  - bag/case_material
+  - color
+  - pattern
+- id: sg-4-1-10-4-2
+  name: Wingfoil Protective Gear
+  children: []
+  attributes: []
+- id: sg-4-1-10-5
+  name: Wingfoil Boards
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-11
+  name: Windfoil
+  children:
+  - sg-4-1-11-1
+  - sg-4-1-11-2
+  - sg-4-1-11-3
+  - sg-4-1-11-4
+  - sg-4-1-11-5
+  attributes: []
+- id: sg-4-1-11-1
+  name: Windfoil Boards
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-11-2
+  name: Windfoil Foils
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-11-3
+  name: Windfoil Sails
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+  - recommended_skill_level
+- id: sg-4-1-11-4
+  name: Windfoil Parts
+  children:
+  - sg-4-1-11-4-1
+  - sg-4-1-11-4-2
+  - sg-4-1-11-4-3
+  - sg-4-1-11-4-4
+  attributes: []
+- id: sg-4-1-11-4-1
+  name: Foil Masts
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+- id: sg-4-1-11-4-2
+  name: Foil Fuselages
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
+- id: sg-4-1-11-4-3
+  name: Foil Wings
+  children: []
+  attributes:
+  - color
+  - pattern
+  - material
```

Comment:
If I'm not mistaken, sails, not wings, typically apply to windfoiling, right?

![image](https://github.com/user-attachments/assets/8a2b3f5f-84cd-4dda-addd-44dbcf9e76b1)